third_party/build/bin/freediameter_build.sh

```diff
@@ -61,7 +61,7 @@ git checkout "${GIT_COMMIT}"
 
 patch -p1 < $MAGMA_ROOT/lte/gateway/c/oai/patches/0001-opencoord.org.freeDiameter.patch
 
-awk '{if (/^DISABLE_SCTP/) gsub(/OFF/, "ON"); print}' CMakeCache.txt > tmp && mv tmp CMakeCache.txt
+awk '{if (/^DISABLE_SCTP/) gsub(/OFF/, "ON"); print}' build/CMakeCache.txt > tmp && mv tmp build/CMakeCache.txt
 
```

Comment:
why are these build script changed? containerized AGW can use already built packages for these dependency

Reply:
I think it was just a nit by ramon. fixed.